### ISSUE TITLE
Error in csv export - table aliases

### ISSUE THREAD

**Issue by LaySoft** [phpmyadmin/phpmyadmin#14612]
phpMyAdmin: 4.8.3, 5.0.0-dev

A have a query with one table two times, but different aliases:
```sql
SELECT u1.alkid, u2.alkid FROM a_ugyanazok u1, a_ugyanazok u2 WHERE u1.groupid = u2.groupid LIMIT 10;
```

If i want to export the result in csv, under Query results operations, the csv file contains the following error:

```sql
SELECT a_ugyanazok.alkid, a_ugyanazok.alkid FROM `a_ugyanazok`, `a_ugyanazok` WHERE a_ugyanazok.groupid = a_ugyanazok.groupid;
```

`#1066 - Not unique table/alias: 'a_ugyanazok'`

**Comment by kartik1000** [comment]
@williamdes, Hey can't we refer this issue to #14691 because both seem have to have same encoding issue.

**Comment by williamdes** [comment]
@kartik1000 Can you please fix this issue ?

**Comment by kartik1000** [comment]
@williamdes, I am onto it

**Comment by williamdes** [comment]
Duplicate of #12241

**Comment by williamdes** [comment]
Will be fixed as #12241
